docs/data/sources/google-tag-manager-client.md

```diff
@@ -8,8 +8,12 @@ This is the client-side Google Tag Manager Template for Amplitude Analytics. The
 !!!info Resources
     [:simple-googletagmanager: GTM Template Gallery](https://tagmanager.google.com/gallery/#/owners/amplitude/templates/amplitude-browser-sdk-gtm-template) · [:material-github: GitHub](https://github.com/amplitude/amplitude-browser-sdk-gtm-template)
 
-With this template, you can defer the `init` tag until you receive a signal, like a consent grant. It's also common to fire the `init` tag using `All Pages` or `Initialization - All Pages` triggers. But make sure to fire the `init` tag **before** any other tags. The legacy template automatically calls init, so there's no need to do it yourself.
+It’s important to create an `init` tag. It's common to fire the `init` tag using `All Pages` or `Initialization - All Pages` triggers. With this template, you can also defer the `init` tag until you receive a signal, such as a consent grant. But notice that all other tags wait for the `init` tag to fire before they can be sent to Amplitude. The legacy template automatically calls init, so there's no need to do it yourself.
```

Comment:
Different from Amplitude Analytics Legacy, in Amplitude Analytics Browser SDK, we separate out the amplitude init call. You must create a separate tag for the `init`.  It's common to fire the `init` tag using `All Pages` or `Initialization - All Pages` triggers.  But you can also fire the init tag based on your customized triggers, such as a consent grant or others to defer the `init` call. But notice that all other tags wait for the `init` tag to fire before they can be sent to Amplitude. The legacy template automatically calls init, so there's no need to do it yourself.

Reply:
Not good at words...  Leave this as a suggestion.

Reply:
Others look good to me.

Reply:
Thanks for your comment! 

I would like to start with the call out to create an `init` tag first as it's the point of this whole paragraph and put legacy SDK at the end. 

Added "customized triggers" to align with the fact that users can actually create customized triggers.